src/steth/scUSDC.sol

```diff
@@ -215,7 +215,9 @@ contract scUSDC is sc4626, IFlashLoanRecipient {
         uint256[] memory amounts = new uint256[](1);
         amounts[0] = debt - wethBalance;
 
+        _initiateFlashLoan();
         balancerVault.flashLoan(address(this), tokens, amounts, abi.encode(collateral, debt));
+        _finalizeFlashLoan();
```

Comment:
```suggestion
        flashLoanInitiated = true;
        balancerVault.flashLoan(address(this), tokens, amounts, abi.encode(collateral, debt));
        flashLoanInitiated = false;
```
cheaper

Reply:
how much cheaper though?

Reply:
<table border="2" cellspacing="0" cellpadding="6" rules="groups" frame="hsides">


<colgroup>
<col  class="org-left" />

<col  class="org-right" />

<col  class="org-right" />

<col  class="org-right" />

<col  class="org-right" />

<col  class="org-right" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-left">Function Name</th>
<th scope="col" class="org-right">min</th>
<th scope="col" class="org-right">avg</th>
<th scope="col" class="org-right">median</th>
<th scope="col" class="org-right">max</th>
<th scope="col" class="org-right"># calls</th>
</tr>
</thead>

<tbody>
<tr>
<td class="org-left">scUSDC receiveFlashLoan</td>
<td class="org-right">2349</td>
<td class="org-right">93607</td>
<td class="org-right">94143</td>
<td class="org-right">183793</td>
<td class="org-right">4</td>
</tr>


<tr>
<td class="org-left">without function call</td>
<td class="org-right">2349</td>
<td class="org-right">93592</td>
<td class="org-right">94119</td>
<td class="org-right">183783</td>
<td class="org-right">4</td>
</tr>


<tr>
<td class="org-left">scWETH receiveFlashLoan</td>
<td class="org-right">2219</td>
<td class="org-right">292628</td>
<td class="org-right">225998</td>
<td class="org-right">488233</td>
<td class="org-right">54</td>
</tr>


<tr>
<td class="org-left">without function call</td>
<td class="org-right">2219</td>
<td class="org-right">293499</td>
<td class="org-right">226020</td>
<td class="org-right">488220</td>
<td class="org-right">54</td>
</tr>
</tbody>
</table>

its negligible i suppose we can keep it as is